hyph-utf8/tex/generic/hyph-utf8/patterns/tex/hyph-mn-cyrl-x-lmc.tex

```diff
@@ -1,17 +1,13 @@
-% This file has been converted for the hyph-utf8 project from mnhyphen.tex
-% (2002-06-30), whose authors have been identified as Oliver Corff <corff at
-% zedat dot fu-berlin dot de> and Dorjpalam Dorj.  The licence terms are unchanged.
-%
-% See http://www.hyphenation.org for details on the project.
-%------------------------------------------------------------------------------
-%
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-%        File: mnhyphen.tex
-%      Author: Oliver Corff and Dorjpalam Dorj
-%        Date: February 26th, 1999              % mls.sty prevails
-%     Version: \VersionRelease                  % see mls.sty!
-%   Copyright: Ulaanbaatar, Beijing, Berlin
-%
+% title: Alternative Hyphenation Patterns for Xalx Mongolianj Cyrillic script
```

Comment:
I guess this is typo and should be "Mongolian" rather than "Mongolianj"?

Reply:
Probably ;-)